common/game_concepts/wc_magic_lifestyles_concepts.txt

```diff
@@ -0,0 +1,68 @@
+﻿wc_magic_lifestyle = {
```

Comment:
Will you chance the lifestyle screen? Right now you can only see the first 5 lifestyles. Adding a scrolling bar or something would imporve it IMO

![bild](https://user-images.githubusercontent.com/4825809/114296488-eb23a900-9aab-11eb-9ead-19313436a3d4.png)

Reply:
You could scroll it before, but I added scrollbar anyway.

Reply:
Good work on this one